README.md

```diff
@@ -59,6 +59,8 @@ Requirement: `npm install nyc`
 | --coverage, --cov          | Enable coverage measurement                           | `boolean` | `false`                         |
 | --coverage-instrument-spec | Coverage instrument pattern from `root`               | `glob`    | `src/**/*.js`                   |
 | --coverage-output-dir      | Coverage output directory in `root`                   | `string`  | `.nyc_output`                   |
+| --clear-coverage-dir,      | Clear the Coverage output directory before running    |           |                                 |
+| --clr-cov                  | test, when coverage option is enabled                 | `bool`    | `true`                          |
```

Comment:
I'm ok with the option name  `clear-coverage-dir` but `--clr-cov` looks weird to me. The long form can be sufficient no ?

Reply:
Sure, we can remove the alias. Just wanted to use a short version because of the long command line in Depp Teachers Portal.

Reply:
Done.